unix/w0vncserver/PipeWirePixelBuffer.cxx

```diff
@@ -94,6 +95,10 @@ void PipeWirePixelBuffer::processFrame(spa_buffer* buffer)
   spa_chunk* chunk;
   uint8_t* dstBuffer;
   pixman_bool_t ret;
+  int x;
+  int y;
+  int w;
+  int h;
```

Comment:
Might want to use a `Rect`, since that is what you commonly create out of these anyway.